x-pack/plugins/upgrade_assistant/__jest__/client_integration/app/cluster_upgrade.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('Cluster upgrade', () => {
 
   describe('when cluster is in the process of a rolling upgrade', () => {
     beforeEach(async () => {
-      httpRequestsMockHelpers.setLoadDeprecationLoggingResponse(undefined, {
+      httpRequestsMockHelpers.setLoadEsDeprecationsResponse(undefined, {
```

Comment:
since the `deprecationLoggingResponse` has now been moved to another page, we need to change the API that we are mocking the error from in order to trigger the cluster upgrading error.

Reply:
I am not sure I follow. How can we interchangeably use  `setLoadEsDeprecationsResponse` and `setLoadDeprecationLoggingResponse` with the same error object?
Both endpoint can return a `426` and an `attributes` prop in the response?

I am not seeing the related change in `app.tsx`, I am probably missing something?

Reply:
The way we detect if we are currently upgrading or if the upgrade has been completed is if we ever get back a 426 error in any api response that UA makes. Previously this test was mocking that with the api that returns the status of the toggle but since that api is no longer being called since it was moved to another page, we can safely just mock any other api that is being called (in this case I just picked the one that returns es stats).

Reply:
I see. It might be good to add a comment for that in the test. No need to do it in this PR to not trigger another CI cycle but let's do it on the feature branch directly after we merge.